src/Nethermind/Nethermind.Merge.Plugin/Synchronization/MergeBlockDownloader.cs

```diff
@@ -172,6 +172,11 @@ bool HasMoreToSync()
                     Block currentBlock = blocks[blockIndex];
                     if (_logger.IsTrace) _logger.Trace($"Received {currentBlock} from {bestPeer}");
 
+                    if (currentBlock.Header.HasBody && currentBlock.Body.IsEmpty)
```

Comment:
This is now duplicated in 3 places, should we have some kind of method for this validation?

Reply:
Probably, I'm still good with this expression IMO, still understandable.

Reply:
Renamed to `currentBlock.IsBodyMissing`